api/tests/integration/custom_auth/end_to_end/test_custom_auth_integration.py

```diff
@@ -701,3 +702,31 @@ def test_cannot_create_superuser_if_any_user_exists(
         "A superuser can only be created through this  endpoint if no other users exist."
     ]
     assert FFAdminUser.objects.filter(email=email).exists() is False
+
+
+@pytest.mark.parametrize("marketing_consent_given", (None, True, False))
+def test_marketing_consent_given_defaults_to_false(
```

Comment:
🤔

Reply:
Fixed in https://github.com/Flagsmith/flagsmith/pull/5416/commits/9da5574633b7c1f887532c9f6c64cb11a5cf70ef